src/transport_ofi.c

```diff
@@ -1007,9 +1040,8 @@ static int shmem_transport_ofi_ctx_init(shmem_transport_ctx_t *ctx, int id)
 
     struct fabric_info* info = &shmem_transport_ofi_info;
     info->p_info->ep_attr->tx_ctx_cnt = FI_SHARED_CONTEXT;
-    info->p_info->caps = FI_RMA | FI_WRITE | FI_READ | FI_ATOMICS
-                         | FI_REMOTE_WRITE | FI_REMOTE_READ;
-    info->p_info->tx_attr->op_flags = FI_DELIVERY_COMPLETE | FI_INJECT_COMPLETE;
+    info->p_info->caps = FI_RMA | FI_WRITE | FI_READ | FI_ATOMICS;
+    info->p_info->tx_attr->op_flags = FI_DELIVERY_COMPLETE;
```

Comment:
Just re-read the descriptions for these flags, and I'm scratching my head.  I think I'd like to understand this field better.  I'm wondering if ```FI_INJECT_COMPLETE``` (which seems too relaxed to support shmem_quiet) combined with bounce buffering was giving us an ordering issue that we were only seeing in the Cray tests?

Reply:
Hmm... could be.  You mean the cswap Cray tests?  I haven't seen the bug yet, we'll have more stats with my next push(es).

Reply:
```FI_DELIVERY_COMPLETE``` is correct and is what we're using on master.  Looks like this oddity might have come over from the prototype implementation.

Reply:
Oh right, I knew that.  So this can't be the cswap problem...